.github/workflows/release.yml

```diff
@@ -26,4 +26,4 @@ jobs:
             AUTOMATED_MAVEN_RELEASE_SONATYPE_PASSWORD: ${{ secrets.AUTOMATED_MAVEN_RELEASE_SONATYPE_PASSWORD }}
           run: |
             cd android
-            ./gradlew :source:publishAllPublicationsToSonatypeRepository
\ No newline at end of file
+            ./gradlew :source:publishToSonatype closeAndReleaseSonatypeStagingRepository
```

Comment:
Using the nexus publish plugin, we first publish the libraries, then ask sonatype to close the staging, and release it to publishing.